### PR TITLE
npchighlight: Improve true locations

### PR DESCRIPTION
**Pull request by Richardant**
Added thin true locations and now marks a single square rather than multiple squares when a npc have a size superior than one

### CODE REVIEW

File: npchighlight/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java [hunk 1]
```diff
@@ -218,17 +218,28 @@ private void renderNpcOverlay(Graphics2D graphics, NPC actor, Color color)
...
 				{
 					size = composition.getSize();
 				}
-
-				final WorldPoint wp = actor.getWorldLocation();
-				final Color squareColor = color;
-
-				getSquare(wp, size).forEach(square ->
-					drawTile(graphics, square, squareColor, 1, 255, 50));
+				LocalPoint localPoint = LocalPoint.fromWorld(this.client, actor.getWorldLocation());
```
Comment (Lucwousin) [inline npchighlight/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java:225]: `LocalPoint.fromWorld` is nullable

Comment (Richardant) [reply npchighlight/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java:225]: hmm doesn't work properly without that

Comment (Lucwousin) [reply npchighlight/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java:225]: I'm telling you to null check it

File: npchighlight/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java [hunk 2]
```diff
@@ -218,18 +218,35 @@ private void renderNpcOverlay(Graphics2D graphics, NPC actor, Color color)
...
-				getSquare(wp, size).forEach(square ->
-					drawTile(graphics, square, squareColor, 1, 255, 50));
-				break;
+				LocalPoint lp = LocalPoint.fromWorld(client, actor.getWorldLocation());
+				if (lp != null)
+				{
+					lp = new LocalPoint(lp.getX() + size * 128 / 2 - 64, lp.getY() + size * 128 / 2 - 64);
+					final Polygon tilePoly = Perspective.getCanvasTileAreaPoly(client, lp, size);
+					renderPoly(graphics, color, tilePoly);
+					break;
```
Comment (Lucwousin) [inline npchighlight/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java:231]: get the `break`'s out of the if statements. this way they fall through

Comment (Richardant) [reply npchighlight/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java]: Done

**Pull request merged**